requirements.txt

```diff
@@ -1,6 +1,6 @@
 joblib>=0.11
 Cython>=0.29,!=0.29.18
-numpy~=1.19.0
+numpy>=1.19.0
```

Comment:
Same here

Reply:
@odidev looks like this file didn't get reverted in your most recent change. Once you revert, I can merge

Reply:
Done.